docs/reference/query-dsl/text-expansion-query.asciidoc

```diff
@@ -155,47 +155,54 @@ GET my-index/_search
 ----
 // TEST[skip: TBD]
 
-This can also be achieved by using sub searches combined with <<rrf>>.
+This can also be achieved by using an <<rrf-retriever, `rrf` retriever>>
+with multiple <<standard-retriever, `standard` retrievers>>.
```

Comment:
It would be nice if we continued to link to the main RRF page here as well. 

```suggestion
This can also be achieved using <<rrf>>, through an <<rrf-retriever, `rrf` retriever>>
with multiple <<standard-retriever, `standard` retrievers>>.
```

Reply:
Changed.